WordPress/src/main/java/org/wordpress/android/ui/media/MediaSettingsActivity.java

```diff
@@ -264,12 +316,22 @@ public void onClick(View v) {
         }
     }
 
+    private boolean isMediaFromEditor() {
+        return mEditorImageMetaData != null;
+    }
+
     private void reloadMedia() {
         loadMediaId(mMedia.getId());
     }
 
     private boolean loadMediaId(int mediaId) {
-        MediaModel media = mMediaStore.getMediaWithLocalId(mediaId);
+        MediaModel media;
+        if (isMediaFromEditor()) {
```

Comment:
This method signature seems off to me, since the passed `mediaId` is ignored when the media came from the editor. Any way we can change this without over-complicating things?